services/movie_session.py

```diff
@@ -4,7 +4,7 @@
 
 
 def create_movie_session(
-    movie_show_time: str, movie_id: int, cinema_hall_id: int
+        movie_show_time: str, movie_id: int, cinema_hall_id: int
```

Comment:
you should pay attention when you use tools for formatting code now indentation is broken everywhere